clients/fides-js/src/components/ConsentModal.tsx

```diff
@@ -48,7 +48,7 @@ const ConsentModal = ({
           i18n={i18n}
           renderModalFooter={renderModalFooter}
         >
-          {renderModalContent()}
+          {children}
```

Comment:
ah, we were already passing `children`, and duplicating it here I see. Thanks for fixing this!